Index.html

```diff
@@ -44,14 +44,174 @@ <h1 >I’m David. Glad to see you!</h1>
+              <li><span><a href="#">Bank End Dev</a></span></li> 
+              <li><span><a href="#">2015</a></span></li>                        
+           </ul>
+      </span>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <span class="sublinksround">          
+          <ul>
+              <li><a href="#">HTML</a></li> 
+              <li><a href="#">CSS</a></li> 
+              <li><a href="#">JavaScript</a></li>          
+           </ul>
+      </span>    
+    <button type="submit" class="call-to-action">See Project</button>
+  </section>  
+  <section class="section1">       
+    <span><img  class="imgalin" src="./images/Snapshoot2.svg"></span>
+    <h2>Multi-Post Stories</h2>
+    <span class="sublinks">
+        <span><b>CANOPY</b></span>
+        <ul>
+            <li><span><a href="#">Bank End Dev</a></span></li> 
+            <li><span><a href="#">2015</a></span></li>                      
+         </ul>
+    </span>
+
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <span class="sublinksround">
+        
+        <ul>
+            <li><a href="#">HTML</a></li> 
+            <li><a href="#">CSS</a></li> 
+            <li><a href="#">JavaScript</a></li>          
+         </ul>
+    </span>    
+  <button type="submit" class="call-to-action">See Project</button>
+ </section>
+ <section class="section1">       
+  <span><img  class="imgalin" src="./images/Snapshoot3.svg"></span>
+  <h2>Tonic</h2>
+  <span class="sublinks">
+      <span><b>CANOPY</b></span>
+      <ul>
+          <li><span><a href="#">Bank End Dev</a></span></li> 
+          <li><span><a href="#">2015</a></span></li>                    
+       </ul>
+  </span>
+
+  <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+  <span class="sublinksround">
+      
+      <ul>
+          <li><a href="#">HTML</a></li> 
+          <li><a href="#">CSS</a></li> 
+          <li><a href="#">JavaScript</a></li>          
+       </ul>
+  </span>    
+  <button type="submit" class="call-to-action">See Project</button>
+</section>
+<section class="section1">       
+  <span><img  class="imgalin" src="./images/Snapshoot4.svg"></span>
+  <h2>Multi-Post Stories</h2>
+  <span class="sublinks">
+      <span><b>CANOPY</b></span>
+      <ul>
+          <li><span><a href="#">Bank End Dev</a></span></li> 
+          <li><span><a href="#">2015</a></span></li>                    
+       </ul>
+  </span>
+
+  <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+  <span class="sublinksround">
+      
+      <ul>
+          <li><a href="#">HTML</a></li> 
+          <li><a href="#">CSS</a></li> 
+          <li><a href="#">JavaScript</a></li>          
+       </ul>
+  </span>    
+  <button type="submit" class="call-to-action">See Project</button>
+  </section>
+  <section class="about">   
```

Comment:
- [ ] KIndly pay close attention to the background of the about section. please note that there is no padding around the about me section. Kindly check the figma design 😄 .

FIgma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/228974000-17e5c3d6-3f71-473c-a1da-4518abd20ac5.png)

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/228974138-3495d2ce-a17e-43eb-b995-feaa36c333e0.png)